package.json

```diff
@@ -53,7 +53,7 @@
   },
   "scripts": {
     "install": "node lib/checkVersions.js",
-    "postinstall_disabled": "node lib/install.js",
+    "postinstall": "node lib/install.js",
```

Comment:
I'm not sure when and why this was changed. I guess we need to do some manual tests in VMs to ensure this is working as intended.

Reply:
Ok I found the change:
https://github.com/ioBroker/ioBroker/commit/8ca35b944d411a78116460b31dffc9418d749576

😐

Reply:
Guess the postinstall script broke Bluefox's deploy script. I think the clean way would be enabling the script and using an environment variable in the deploy step to exit it early instead.

Reply:
I asked for reason ...

Reply:
The deploy script needs `gulp` to be installed, but the postinstall script will fail the installation (because it is not meant to be npm installed outside the installer script).
IMO the clean way would be removing gulp from the dependencies, and just executing it during the deploy step e.g. with `npx gulp`. No need to fiddle with dependencies and scripts.

Reply:
@mfechner I'll prepare a commit with what I think is correct

Reply:
Cannot push to your PR branch somehow, here's the patch file

```patch
From a3aa192e824463d5b7e793e8777c19651bcc822b Mon Sep 17 00:00:00 2001
From: Dominic Griesel <d.griesel@gmx.net>
Date: Wed, 2 Dec 2020 11:02:32 +0100
Subject: [PATCH] skip postinstall script during the deploy action

---
 .github/workflows/deploy.yml | 1 +
 lib/install.js               | 6 ++++++
 2 files changed, 7 insertions(+)

diff --git a/.github/workflows/deploy.yml b/.github/workflows/deploy.yml
index 1d01bc8..0c67f07 100644
--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -15,6 +15,7 @@ jobs:
       - run: npm i
       - run: npm run deploy
         env:
+          SKIP_POSTINSTALL: true
           SFTP_HOST: ${{secrets.SFTP_HOST}}
           SFTP_PASS: ${{secrets.SFTP_PASS}}
           SFTP_PORT: ${{secrets.SFTP_PORT}}
diff --git a/lib/install.js b/lib/install.js
index a783913..81abd89 100644
--- a/lib/install.js
+++ b/lib/install.js
@@ -1,5 +1,11 @@
 'use strict';
 
+// If this script is run in a context where ioBroker itself shouldn't be installed,
+// then skip this script
+if (!!process.env.SKIP_POSTINSTALL) {
+    process.exit(0);
+}
+
 const tools = require('./tools.js');
 const platform = require('os').platform();
 const colors = require('colors');
-- 
2.28.0.windows.1
```

Reply:
I can disable it again, but this will cause the Cirrus run here to fail, what you prefer.
Maybe you modify this later if you publish the package to npmjs.org.

Reply:
No, don't disable it. My patch makes it so that only the deploy job skips the postinstall script instead of disabling the script altogether.

Reply:
My commit is meant to be applied after your changes.

Reply:
I added it.